.mergify.yml

```
@@ -0,0 +1,19 @@
+pull_request_rules:
+  - name: backport to master
+    conditions:
+      - merged
+      - base=dev
+      - label="please backport"
+    actions:
+      backport:
+        branches:
+          - master
+        ignore_conflicts: True
+        label_conflicts: "bp-conflict"
+
+  - name: label Mergify backport PR
+    conditions:
+      - body~=This is an automated backport of pull request \#\d+ done by Mergify
+    actions:
+      label:
+        add: [backported]
```

```suggestion
        add: [backport]
```

This goes on the PR to master

---

I was worried users would apply this to their dev branch since the label is what triggers the BP here versus the milestone. Maybe i'll change it to "is backport".